npm-shrinkwrap.json

```diff
@@ -1271,56 +4276,274 @@
       "from": "ms@0.7.2",
       "resolved": "https://registry.npmjs.org/ms/-/ms-0.7.2.tgz"
     },
+    "murmur-32": {
+      "version": "0.1.0",
+      "from": "murmur-32@>=0.1.0 <0.2.0",
+      "resolved": "https://registry.npmjs.org/murmur-32/-/murmur-32-0.1.0.tgz"
+    },
     "mute-stream": {
       "version": "0.0.5",
       "from": "mute-stream@0.0.5",
       "resolved": "https://registry.npmjs.org/mute-stream/-/mute-stream-0.0.5.tgz"
     },
-    "nan": {
```

Comment:
I was concerned that some modules have been removed from the shrinkwrap, but it's actually just github's diff-display getting confused ;-)